frontend/src/components/common/LogViewer.tsx

```diff
@@ -148,7 +148,7 @@ export function LogViewer(props: LogViewerProps) {
             width: '100%',
             height: '100%',
             '& .terminal.xterm': {
-              padding: 10,
+              padding: '10px',
```

Comment:
I think we should avoid pixel units for padding/spacing but instead use the `theme.spacing()` function. Can you try with `theme.spacing(1)`?